src/Mailer/Command/SendOwnershipRequestedMail.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer\Command;
+
+use CultuurNet\UDB3\CommandHandling\AsyncCommand;
+use CultuurNet\UDB3\CommandHandling\AsyncCommandTrait;
+
+final class SendOwnershipRequestedMail implements AsyncCommand
+{
+    use AsyncCommandTrait;
+
+    private string $uuid;
+
+    public function __construct(string $uuid)
```

Comment:
All the existing ownership commands are typed on `Uuid` so I think that will also work here?

Reply:
I did this because this command will be serialized on a message queue, and I was thought that you want to use primitive types to not use to much memory/storage on the queue system, in this case reddis.
What is your opinion on this?